tests/specs/metrics.e2e.js

```diff
@@ -357,15 +200,15 @@ function loaderTypeSupportabilityMetric (loaderType) {
 
     const supportabilityMetrics = unloadSupportMetricsResults.request.body.sm
     expect(supportabilityMetrics).toEqual(expect.arrayContaining([{
-      params: { name: expect.stringContaining('Generic/Version/') },
+      params: { name: expect.stringContaining('Generic/DistMethod/') },
       stats: { c: 1 }
     }]))
     expect(supportabilityMetrics).toEqual(expect.arrayContaining([{
-      params: { name: expect.stringContaining('Generic/DistMethod/') },
+      params: { name: `Generic/LoaderType/${loaderTypesMapped[loaderType]}/Detected` },
```

Comment:
Does this test account for the worker loader type?

Reply:
No. It would take more work to support worker here and I am not sure that work is worth it right now.

Reply:
No. It would take more work to support worker here and I am not sure that work is worth it right now.